android/gradle.properties

```
@@ -10,7 +10,7 @@
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
 # Default value: -Xmx512m -XX:MaxMetaspaceSize=256m
-org.gradle.jvmargs=-Xmx2048m -XX:MaxMetaspaceSize=512m
+org.gradle.jvmargs=-Xmx4608m -XX:MaxMetaspaceSize=512m
```

Should it be this instead - times 2?
```suggestion
org.gradle.jvmargs=-Xmx4096m -XX:MaxMetaspaceSize=512m
```

---

Are you telling me `2048 * 2 != 4608` !!

Yeah, good spot